### PR TITLE
22 cli: add heurist option to llm providers

### PR DESCRIPTION
**Pull request by cristiam86**
Added Heurist provider

### CODE REVIEW

File: src/lib/config/simulator.ts [hunk 1]
```diff
@@ -15,13 +15,25 @@ export type RunningPlatform = (typeof AVAILABLE_PLATFORMS)[number];
...
+  openai: {
+    name: "OpenAI (You will need to provide an OpenAI API key)",
+    envVar: "OPENAIKEY",
+    cliOptionValue: "openai",
+  },
+  heurist: {
+    name: "Heurist (You will need to provide an API key)",
+    envVar: "HEURISTAIAPIKEY",
+    cliOptionValue: "heurist",
+  },
```
Comment (MuncleUscles) [inline src/lib/config/simulator.ts:38]: I think we should add something here that explains that you can get free credits 

> the automated invite code system is ready https://dev-api-form.heurist.ai/ (accessible from heurist.ai homepage) you can try it out and fill in "genlayer" in the last question

Comment (cristiam86) [reply src/lib/config/simulator.ts:38]: what about:
- name: "Heurist (You will need to provide an API key which you can get for free at https://dev-api-form.heurist.ai by entering the referral code: genlayer)"

Comment (MuncleUscles) [reply src/lib/config/simulator.ts:38]: > Heurist (You will need to provide an API key. Get free API credits at https://dev-api-form.heurist.ai/ with referral code: "genlayer"):

**Review by MuncleUscles — APPROVED** [review]
Ready to go otherwise

**Review by MuncleUscles — APPROVED** [review]
Perfect

**Pull request merged**